src/style.css

```
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  max-width: 100%;
+  border-radius: 4px;
+}
+
+.big {
+  font-family: Avenir, Helvetica, sans-serif;
```

Specify font-family in `html` tag selector 
```suggestion
  font-family: inherit; 
```